fichier_r.qmd

```diff
@@ -1,178 +1,169 @@
--   the meteorological data of French Guiana from 1950 to 2024\
--   the capture locations coordinates (degree minute second)
+French Guiana is a territory belonging to the bio-geographic unit of the Guiana Shield (@hollowell_checklist_2005) With an area of 83,846 km², it hosts, like its neighboring territories, remarkable species diversity. Covered by 97% forest habitat mosaic, its extreme species diversity is well-documented. French Guiana represents a global hotspot in terms of bat species richness. The most commonly used method for studying bats in tropical regions is ground-level mist netting, followed by canopy netting and direct roost searche (@macswiney_g_what_2008). It is with these methods that the first studies on bat community characterization were conducted (@charles-dominique_pierre_chauves-souris_2001). [To date, very little work has focused on exploring the reproduction patterns of Neotropical bat communities. Reproductive patterns are closely related to rainfall regimes (@ruiz-ramoni_reproductive_2016) frequently occurring twice a year in tropical regions.]{.underline}
 
-```{r}
+-   The data collected on the species of bats captured by date and location in French Guiana, and containing information on sex, age and reproductive state\
+-   The meteorological data of French Guiana from 1950 to 2024\
+-   The capture locations coordinates (degree minute second)
+
+# II. Formating our datasets
 
+## A. Importation of datasets and packages
+
+```{r include=FALSE}
 library(dplyr)
 library(readxl)
 library(lubridate)
+library(ggplot2)
+library(stringr)
+library(tidyr)
+library(R2jags)
+library(ade4)
+library(spdep)
+library(tripack)
+library(sp)
+library(geoR)
+```
 
-data_chiro<-read_excel("data_chiro.xlsx") #import data chiro
-data_abiot_50_22<-read.csv("abiot_1950_2022.csv", sep = ";") #import data meteo 1950 -> 2022 #pensez à mettre les sources
-data_abiot_23_24<-read.csv("abiot_2023_2024.csv", sep = ";") #import data meteo 2023 -> 2024
+```{r include=FALSE}
+data_chiro<-read_excel("data_chiro.xlsx")
+data_abiot_50_22<-read.csv("abiot_1950_2022.csv", sep = ";") 
+data_abiot_23_24<-read.csv("abiot_2023_2024.csv", sep = ";") 
 coord_chiro<-read_excel("COORD_CAPTURE.xlsx")
-
 ```
 
-#### Data processing
+## B. Processing of the abiotic dataset
 
 -   Firstly we concatenate all the abiotic data together
 
--   Then we start to clean the abiotic the dataset by deleting all years that precede the first event of capture, the we use the Lubridate R package [@grolemund2016lubridate] to formate the data column as we want and the dplyr R package [@wickham2014dplyr] to re-organize the data set
+-   Then we start to clean the abiotic the dataset by deleting all years that precede the first event of capture, the we use the Lubridate R package (@spinu_lubridate_2010) to formate the data column as we want and the dplyr R package (@wickham_dplyr_2023) to reorganize the data set.
 
 ```{r}
```

Comment:
Maybe you could add titles to your chunks to make them more precise and avoid losing your readers.